First Task/.gitignore_global

```diff
@@ -0,0 +1,6 @@
+.idea/
+.gradle/
+out/
+build/
```

Comment:
`.gitignore` in the project root better leave for project-generated files. Those are a pretty rare case. And **global** `.gitignore` is a file that you use for all the projects you are working on. It is usually is placed in your user home directory (as you might know the `~` symbol on all the Unix-based systems resolves as a current user home directory). Let's make it together.

**Steps**
1) Check if you already have **global** `.gitignore` file
```bash
test -f ~/.gitignore && echo "Found: skip the next step" || echo "Not found: move on to the next step"
```
2) Create `.gitignore` file in your user home directory:
```bash
touch ~/.gitignore
```
3) Add patterns you want git to ignore globally, at least the following ones:
```bash
cat >> ~/.gitignore << EOL
.idea/
.gradle/
out/
build/
target/
*.iml
EOL
```
4) And then say `git` to use this file:
```bash
git config --global core.excludesfile ~/.gitignore
```